docs/user-guide/creating-application/docker-build-configuration.md

```diff
@@ -25,6 +25,13 @@ Select the docker registry that you wish to use. This registry will be used to [
 ### 2. Docker Repository
 In this field, add the name of your docker repository. The repository that you specify here will store a collection of related docker images. Whenever an image is added here, it will be stored with a new tag version.
 
+**If you are using docker hub account, you need to enter the repository name along with your username.**
```

Comment:
rephrase the sentence and explain it with examples like for repo `abc/def`  enter   `abc/def` not just  `def`